integration_tests/mockApis/prison.ts

```diff
@@ -33,6 +33,40 @@ export default {
       },
     }),
 
+  stubOffenceCode: (offenceCode: OffenceDto): SuperAgentRequest =>
+    stubFor({
+      request: {
+        method: 'GET',
+        url: prisonApiPaths.offenceCode({ offenceCode: offenceCode.code }),
+      },
+      response: {
+        headers: { 'Content-Type': 'application/json;charset=UTF-8' },
+        jsonBody: {
+          content: [offenceCode],
+        },
+      },
+    }),
```

Comment:
This is less complicated than the one in the `wiremock` folder - is the rest not needed here?